kappa/event_source.py

```diff
@@ -73,12 +75,37 @@ def add(self, function):
         except Exception:
             LOG.exception('Unable to add event source')
 
+    def enable(self, function):
+        self.enabled = True
+        try:
+            response = self._lambda.call(
+                'update_event_source_mapping',
+                FunctionName=function.name,
+                Enabled=self.enabled
+            )
+            LOG.debug(response)
+        except Exception:
+            LOG.exception('Unable to enable event source')
+
+    def disable(self, function):
```

Comment:
i'd consider just a set_enabled(self, function, enabled=True) to reduce the duplication.